documentation/binding-data/tutorial-flow-components-binder-validation.asciidoc

```diff
@@ -149,13 +169,24 @@ binder.forField(marriedField)
         .bind(Person::getMaritalStatus, Person::setMaritalStatus);
 ----
 
-Multiple validators and converters can be used for building one binding.
-Each validator or converter is used in the order they were defined for a value provided by the user.
-The value is passed along until a final converted value is stored in the business object, or until the first validation error or impossible conversion is encountered.
-When updating the UI components, values from the business object are passed through each converter in the reverse order without doing any validation.
+=== Adding Multiple Converters
+
+You can add multiple converters (and validators) for each binding.
+
+Each validator or converter is used in the order defined for a value to be input by the user. The value is passed along until:
+
+* A final converted value is stored in the business object, or 
+* The first validation error or impossible conversion is encountered.
+
+When updating UI components, values from the business object are passed through each converter in reverse order (without validation).
+
+//What does "Each validator or converter is used in the order defined for a value to be input by the user"? The field order or the definition order or something else?
```

Comment:
@ruthstocks I think this refers to the order of definition (of validators and converters). The sentence is quite odd, so if you can come up with improvements here, it would be really valuable...

Reply:
Now that I looked the original text, I think this explanation is related to the next code snippet. The subtitle which is converted from a [NOTE] kind of breaks the flow here now.

Reply:
I've updated. Please have a look and let me know.